### PR TITLE
[5.1] Use Dialog for extensions changelog

### PR DESCRIPTION
**Pull request by Fedik**
### Summary of Changes

Use Dialog for extension changelog

### Testing Instructions

You need an extension with changelog.
Open Extensions => Manage, find the extension with changelog and click changelog btn.
Open Update overview and click on the extension changelog btn, if there any update :)

Also can use Test instruction from https://github.com/joomla/joomla-cms/pull/41675

### Actual result BEFORE applying this Pull Request
Works with BS Modal

### Expected result AFTER applying this Pull Request
Works with Joomla Dialog

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed
- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

References:
- https://github.com/joomla/joomla-cms/pull/40150

### CODE REVIEW

File: administrator/components/com_installer/tmpl/manage/default.php [hunk 1]
```diff
@@ -118,20 +118,18 @@
...
+                                                'popupType'  => 'ajax',
+                                                'textHeader' => Text::sprintf('COM_INSTALLER_CHANGELOG_TITLE', $item->name, $item->version),
+                                                'src'        => Route::_('index.php?option=com_installer&task=manage.loadChangelogRaw&eid=' . $item->extension_id . '&source=manage&format=raw', false),
+                                                'width'      => '800px',
+                                                'height'     => 'fit-content',
+                                            ];
                                             ?>
+                                            <button type="button" class="btn btn-info btn-sm"
+                                                data-joomla-dialog="<?php echo $this->escape(json_encode($popupOptions, JSON_UNESCAPED_SLASHES)); ?>">
+                                                <?php echo $item->version?></button>
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/manage/default.php]:
```suggestion
                                                <?php echo $item->version; ?></button>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 46f3b9cbd74c087c4b9db65e331621a0bae04687<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42453">issues.joomla.org/tracker/joomla-cms/42453</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 2b378a1d541c305384166e28d9d1525563e79cc0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42453">issues.joomla.org/tracker/joomla-cms/42453</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42453">issues.joomla.org/tracker/joomla-cms/42453</a>.</sub>

**Comment by LadySolveig** [comment]
Thank you @Fedik and for testing @Quy @viocassel

**Pull request merged**